40K/magazine.json

```diff
@@ -104,7 +104,7 @@
 	"material" : "steel",
 	"symbol" : "#",
 	"color" : "light_gray",
-	"ammo_type" : "bolt",
+	"ammo_type" : "75bolt",
```

Comment:
Everything looks good. I'm going to push this in a little bit so make any changes that you need to.